src/main/java/lotto/StringAddCalculator.java

```diff
@@ -0,0 +1,43 @@
+package lotto;
+
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+	private static String DEFAULT_DELIMITER = ",|:";
+
+	public static int splitAndSum(String inputText) {
+		inputText = Optional.ofNullable(inputText).map(String::valueOf).orElse("");
+		Matcher m = Pattern.compile("//(.)\n(.*)").matcher(inputText);
+		if (m.find()) {
+			return sum(m.group(2).split(m.group(1)));
+		}
+		return sum(inputText.split(DEFAULT_DELIMITER));
+	}
+
+	private static Integer sum(String[] tokens) {
+		Integer result = 0;
+		for (String input : tokens) {
+			result += validateNumber(convertInteger(input));
+		}
+		return result;
+
+	}
+
+	private static Integer convertInteger(String input) {
+		if (input.isEmpty()) {
+			return 0;
+		}
+		return Integer.parseInt(input);
+	}
+
+	private static Integer validateNumber(Integer value) {
```

Comment:
```suggestion
	private static Integer validateNumber(int value) {
```
Integer로 파라미터 받아야할 이유가 있나요?
Integer 타입으로 받으면 null 체크하는 로직도 추가되야 합니다

그리고 메서드명이 적절하지 않습니다.
숫자 유효성 검사가 아닌 양수 여부를 체크하는 기능이니 적절한 메서드명으로 변경해보아요